app/uk/gov/hmrc/customerprofile/controllers/CustomerProfileController.scala

```diff
@@ -38,6 +38,22 @@ trait CustomerProfileController extends HeaderValidator {
 
   def paperlessSettingsOptOut(journeyId: String): Action[AnyContent]
 
+  def preferencesPendingEmail(journeyId: String): Action[JsValue] =
+    withAcceptHeaderValidationAndAuthIfLive().async(controllerComponents.parsers.json) { implicit request =>
+      implicit val hc: HeaderCarrier = fromHeadersAndSession(request.headers, None)
+      request.body
+        .validate[ChangeEmail]
+        .fold(
+          errors => {
+            Logger.warn("Received error with service getPaperlessSettings: " + errors)
```

Comment:
I think this can change to something more like:
`"Errors validating request body: " + errors`